ui/window.go

```diff
@@ -956,8 +972,16 @@ important changes of the last two versions officially released.
 
 [::b]THIS VERSION
 	- Features
-	- Changes
-	- Bugfixes
+[::b]2020-11-16
+	- Features
```

Comment:
There should be one item named `Vim mode` and sub-items should only list necessary infos. The changelog is not documentation (I am not saying we don't need docs)

Also, the `-` starts a new item, not a new line. This is basically plain markdown ^^